expertise/service/config/default.cfg

```diff
@@ -5,3 +5,16 @@ SPECTER_DIR = '../expertise-utils/specter/'
 MFR_VOCAB_DIR = '../expertise-utils/multifacet_recommender/feature_vocab_file'
 MFR_CHECKPOINT_DIR = '../expertise-utils/multifacet_recommender/mfr_model_checkpoint/'
 CELERY_CONFIG = 'expertise.service.config.celery_config'
+DEFAULT_CONFIG = {
+    "dataset": {},
+    "model": "specter+mfr",
+    "model_params": {
+        "use_title": True,
+        "batch_size": 4,
+        "use_abstract": True,
+        "average_score": False,
+        "max_score": True,
+        "skip_specter": False,
+        "use_cuda": False
+    }
```

Comment:
I'm realizing that in the POST request you send these parameters as well and they have underscores. Well, I guess we can leave the change for a different PR.